grc.fraudDetection_web/README.md

```diff
@@ -0,0 +1,143 @@
+# fraudDetection_web
+
+This project uses Quarkus, the Supersonic Subatomic Java Framework in order to let us present results with GraphQL.
+In this project, we use TypeDB to represent a queryable database of relevant fraud detection data about some transactions between Cardholders and Companies.
+
+## Introduction
+
+We have an imaginary transaction dataset and our application implements some very basic functions:
+
+- Search for Bank
+- Search for Cardholder
+- Search for Company
+- Search for Transaction 
+
+We are demonstrating sub-typing, powerful rules and rule combination in our schema. 
+####
+We are also how easy it is to create complex queries using query composition.
+####
+We are using the DAO design pattern, we have following components on which our design depends:
+
+- The model which is transferred from one layer to the other.
+- The interfaces which provides a flexible design.
+- The interface implementation which is a concrete implementation of the persistence logic.
+
+The querry composition can be observed in all DAOs.
+```javascript
+String getQueryStr = "match " + TX_MATCH + CardholderDao.CARDHOLDER_MATCH + BankDao.BANK_MATCH + MerchantDao.MERCHANT_MATCH;
+```
+
+## Running the application in dev mode
+
+1. Checkout this repository: `git clone https://github.com/vaticle/typedb-examples && cd typedb-examples`.
```

Comment:
```suggestion
1. Checkout this repository: `git clone https://github.com/vaticle/typedb-examples && cd typedb-examples/grc.fraudDetection_web`.
```

Reply:
This also raises the fact we should have a better name and structure for where this example fits into the rest of typedb-examples.

This could fit under our existing commerce package? But perhaps a `security` or `cybersecurity` module would be more appropriate.